tests/rule_based_profiler/parameter_builder/test_simple_date_format_string_parameter_builder.py

```diff
@@ -0,0 +1,156 @@
+import pytest
```

Comment:
@alexsherstinsky I added the API unit tests below following our conversations with @Shinnnyshinshin regarding testing strategy.

Reply:
whoohoo. I love it!